python/katana/example_data.py

```diff
@@ -150,6 +159,7 @@ def find_path() -> Path:
         found_path = None
         # If SRC_DIR or KATANA_SOURCE_DIR environment is set, just use it
         env_path = []
+        print(f"os.environ = {os.environ}")
```

Comment:
This shouldn't even be logged like this. Just remove this. If you want to log specific environment variables. That's fine.